src/hiprtc.cpp

```diff
@@ -354,13 +354,17 @@ namespace
 {
     class Unique_temporary_path {
         // DATA
+        std::string defaultName{"/tmp/hipTmpFile_"};
```

Comment:
Not a cross-platform solution.

Reply:
I agree. I feel continuing to use ```tmpnam``` is better.